config/e2e/monitoring.yaml

```diff
@@ -278,6 +286,14 @@ spec:
       hosts: ['${monitoring_url}']
       username: ${monitoring_user}
       password: ${monitoring_pass}
+      # Since E2E cluster's Elasticsearch instance may lag behind the newest version of Beat, we need to allow this scenario.
+      allow_older_versions: true
+      # The maximum size to send in a single Elasticsearch bulk API index request. Needed if Elasticsearch is behind Cloud Ingress.
+      bulk_max_body_size: 10M
+      # The maximum number of events to bulk in a single Elasticsearch bulk API index request. Needed if Elasticsearch is behind Cloud Ingress.
+      bulk_max_size: 10
+      # The maximum byte size to send in a single Elasticsearch bulk API index request, regardless of compression. Needed if Elasticsearch is behind Cloud Ingress.
+      bulk_max_bytes: 10485760
```

Comment:
I struggle to find any reference to these options in Beats documentation with the exception of `allow_older_versions` and `bulk_max_size` The only thing I could find was a user requesting for those to be added, but afaik that never happened. Do they exist?

Reply:
My apologies.  I really should have referenced the [beat configuration reference](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-reference-yml.html) before adding those.  I'm testing this configuration locally now again to ensure I don't see any `413 Request Entity Too Large` errors again.